### PR TITLE
SameSite - IE 11 started supporting it on recent Windows 10 (#12535)

### LINKED ISSUE #12535: http.headers.Set-Cookie.SameSite - IE 11 on
#### What information was incorrect, unhelpful, or incomplete?
SameSite feature available in IE11, we are showing in the table as "RED" i.e. not available. 

#### What did you expect to see?
SameSite feature available in IE11 on latest Windows 10. There is no default enforcing though. 

#### Did you test this? If so, how?
Yes, tested on IE 11 on recently patched Windows 10. 
Microsoft documentation : https://blogs.windows.com/msedgedev/2018/05/17/samesite-cookies-microsoft-edge-internet-explorer/ 
> Same-site cookies are now available for Microsoft Edge and Internet Explorer on the Windows 10 Fall Creators Update and later via the June 2018 Windows Update packages. 

<details>
<summary>MDN page report details</summary>

* Query: `http.headers.Set-Cookie.SameSite`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite
* Report started: 2021-09-22T10:01:33.015Z

</details>

### PR DESCRIPTION
**Pull request by debiru**
#### Summary
IE 11 started supporting it on recent Windows 10.
The "notes" in IE is reprinted from https://caniuse.com/same-site-cookie-attribute

#### Test results and supporting details
It supports only SameSite attribute, SameSite=Lax, SameSite=None and SameSite=Strict. It not supports Defaults to Lax.

see also #12535 and https://caniuse.com/same-site-cookie-attribute

#### Related issues
Fixes #12535

### CODE REVIEW

**Comment by debiru** [comment]
@ddbeck If you don't mind, I'd appreciate it if you could take a look. It would be great if you could merge it or comment.

File: http/headers/set-cookie.json [hunk 1]
```diff
@@ -170,7 +170,12 @@
                 "version_added": "60"
               },
               "ie": {
-                "version_added": false
+                "version_added": "11",
+                "partial_implementation": true,
+                "notes": [
+                  "Not shipped with the inital release but later with the 2018 June security update (Patch Tuesday) to Windows 10 RS3 (2017 Fall Creators Update) and newer. <a href='https://github.com/MicrosoftEdge/Status/issues/616'>More info</a>.",
+                  "Partial support because only supported in IE 11 on Windows 10 RS3 (2017 Fall Creators Update) and newer, but not in IE 11 on other Windows versions (Windows 7, ...)"
+                ]
```
Comment (queengooborg) [inline http/headers/set-cookie.json:178]: I think that we should omit these notes, and just say `"version_added": "11"` (no partial implementation either).  Even though CanIUse is recording this data, I believe that it's more detail than this project is targeting.  This is the same reasoning as why we [only record major and minor version updates but not patch updates in BCD](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#release-lines), as users are most likely to run the latest patch release for their browser

Comment (debiru) [reply http/headers/set-cookie.json:178]: Yes, I understand. I would like to respond.

**Review by queengooborg — APPROVED** [review]
Thank you, LGTM!

**Pull request merged**